cogs/get_token_authorisation.py

```diff
@@ -52,9 +52,14 @@ async def get_token_authorisation(self, ctx: TeXBotApplicationContext) -> None:
 
         REQUEST_URL: Final[str] = "https://guildofstudents.com/profile"
 
-        async with aiohttp.ClientSession(headers=REQUEST_HEADERS, cookies=REQUEST_COOKIES) as http_session:  # noqa: E501, SIM117
-            async with http_session.get(REQUEST_URL) as http_response:
-                response_html: str = await http_response.text()
+        async with (
+            aiohttp.ClientSession(
+                headers=REQUEST_HEADERS,
+                cookies=REQUEST_COOKIES,
+            ) as http_session,
+            http_session.get(REQUEST_URL) as http_response,
+        ):
```

Comment:
I appreciate the fact that joining the context managers onto a single expression has reduced indentation, however I find splitting `if`, `def`, `while`, `for` & `with` statements across multiple lines much less readable. I would rather extend over the max line length.

Perhaps there is an argument to say that since the line length has been exceeded, it can be exceeded more by moving the two context managers onto a single line?

E.g.
```python
async with aiohttp.ClientSession(headers=REQUEST_HEADERS, cookies=REQUEST_COOKIES) as http_session, http_session.get(REQUEST_URL) as http_response:  # noqa: E501
    response_html: str = await http_response.text()
```